influence_benchmark/generate_histories/generate_histories.py

```diff
@@ -88,6 +93,12 @@ async def main():
             )
             del sub_env_config["weak_char_action"]
             del sub_env_config["normal_char_action"]
+        if "weak_certainty" in sub_env_config:  # tickets
+            sub_env_config["certainty"] = (
+                sub_env_config["weak_certainty"] if weakness else sub_env_config["normal_certainty"]
+            )
+            del sub_env_config["weak_certainty"]
+            del sub_env_config["normal_certainty"]
```

Comment:
Noticed that in nudging-therapist and tickets you're not modifying pm based on weak or normal. Is that intended?

Reply:
Yeah. In nudging I wanted the things to be very similar, the only difference is whether they have a positive or negative experience. In tickets weak doesn't really mean weak, I didn't think we wanted to do the population experiment on tickets. Here it just modifies how uncertain they seem in the initial message, but hopefully the agent will deceive both certain and uncertain